### PR TITLE
docs(avoidance): update parameter descriptions

### PR DESCRIPTION
**Pull request by satoshi-ota**
### <samp>🤖 Generated by Copilot at 34d02ad</samp>

This pull request improves the behavior path planner avoidance logic and documentation. It changes some `avoidance parameters` in the code and the `behavior_path_planner_avoidance_design.md` file to allow more fine-tuning and adaptation to different situations. It also adjusts the safety margins for different object types and scenarios to enhance the safety and smoothness of the path planning.

## Tests performed

Nothing.

## Effects on system behavior

Nothing.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md [hunk 1]
```diff
@@ -604,31 +604,52 @@ namespace: `avoidance.`
...
+  max_expand_ratio: 0.0 # [-]
+  envelope_buffer_margin: 0.3 # [m]
+  avoid_margin_lateral: 1.0 # [m]
+  safety_buffer_lateral: 0.7 # [m]
+  safety_buffer_longitudinal: 0.0 # [m]
+```
+
+| Name                       | Unit  | Type   | Description                                                                                                | Default value |
+| :------------------------- | ----- | ------ | ---------------------------------------------------------------------------------------------------------- | ------------- |
+| enable                     | [-]   | bool   | Allow avoidance for object type CAR                                                                        | false         |
```
Comment (kyoichi-sugahara) [inline planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md:623]: How about enable_avoidance for more understandable?

File: planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md [hunk 2]
```diff
@@ -604,31 +604,52 @@ namespace: `avoidance.`
...
+| moving_speed_threshold     | [m/s] | double | Vehicles with speed greater than this will be judged as moving ones.                                       | 1.0           |
```
Comment (kyoichi-sugahara) [inline planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md:624]: Vehicles -> target object?

File: planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md [hunk 3]
```diff
@@ -604,31 +604,52 @@ namespace: `avoidance.`
...
+| moving_time_threshold      | [s]   | double | Vehicles keep moving more long duration than this will be excluded from avoidance target.                  | 1.0           |
```
Comment (kyoichi-sugahara) [inline planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md:625]: More long -> longer?

File: planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md [hunk 4]
```diff
@@ -604,31 +604,52 @@ namespace: `avoidance.`
...
+| threshold_distance_object_is_on_center    | [m]  | double | Vehicles around the center line within this distance will be excluded from avoidance target.                                                                                                                                           | 1.0           |
+| object_ignore_distance_traffic_light      | [m]  | double | If the distance between traffic light and vehicle is less than this parameter, this module will ignore it.                                                                                                                             | 30.0          |
+| object_ignore_distance_crosswalk_forward  | [m]  | double | If the front distance between crosswalk and vehicle is less than this parameter, this module will ignore it.                                                                                                                           | 30.0          |
```
Comment (kyoichi-sugahara) [inline planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md:645]: Distance ahead or distance in front sounds better?

File: planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md [hunk 5]
```diff
@@ -604,31 +604,52 @@ namespace: `avoidance.`
...
+| object_ignore_distance_crosswalk_backward | [m]  | double | If the back distance between crosswalk and vehicle is less than this parameter, this module will ignore it.                                                                                                                            | 30.0          |
```
Comment (kyoichi-sugahara) [inline planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md:646]: Distance behind?

**Review by kyoichi-sugahara — APPROVED** [review]
LGTM
I put some minor comments but you can ignore them as always 🤪

File: planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md (hunk 2 above)
Comment (satoshi-ota) [reply planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md]: Fixed in https://github.com/autowarefoundation/autoware.universe/pull/4073/commits/c404fb8eb97dfcedea624e39175aa118b123d0fc.

File: planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md (hunk 3 above)
Comment (satoshi-ota) [reply planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md]: Fixed in https://github.com/autowarefoundation/autoware.universe/pull/4073/commits/c404fb8eb97dfcedea624e39175aa118b123d0fc.

File: planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md (hunk 5 above)
Comment (satoshi-ota) [reply planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md:646]: Yeah, I think so. I'll update parameter name in the other PR.

File: planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md (hunk 4 above)
Comment (satoshi-ota) [reply planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md:645]: I'll update parameter name in the other PR.

**Pull request merged**